corehq/apps/accounting/tasks.py

```diff
@@ -810,6 +812,44 @@ def calculate_users_in_all_domains(today=None):
     auto_deactivate_mobile_workers.delay()
 
 
+def count_form_submitting_mobile_workers(domain, start, end):
+    """
+    Returns the count of mobile workers who have submitted a form in the time specified.
+    """
+    form_counts_by_worker = (
+        FormES(for_export=True)
+        .domain(domain)
+        .submitted(gte=start, lt=end)
+        .user_type('mobile')
+        .user_aggregation()
+        .size(0)
+        .run()
+        .aggregations.user.normalized_buckets
+    )
+    return len(form_counts_by_worker)
```

Comment:
We still get ids of the forms mapped to each user (from `normalized_buckets`), but it avoids returning the documents themselves. It was modified  from this similar query: https://github.com/dimagi/commcare-hq/blob/bdfe26a9b6881d6801f39c23d6ce54011012c417/corehq/apps/data_analytics/esaccessors.py#L59-L66

Specifically by not fetching `user_ids` first because we can specify the `user_type`, and by just using `normalized_buckets` because `counts_by_bucket` does more work than we needed here. 

There might still be a nicer way to _just_ get the user ids or even the count of users directly, but I'm not aware of one.